### DISCUSSION TITLE
ltsp image error (aborting ltsp)

### DISCUSSION THREAD

**Discussion by Squialz** [ltsp/ltsp#701] [Q&A]
good morning

I hope I'm talking to the right place I have a problem with the ltsp image command I don't understand I followed the doc of (https://ltsp.org/docs/installation/) and (https://ltsp.org/man/ltsp-image/) but it gives me the error in the capture someone has it any solution for me? for more precision I am on proxmox and the .vmdk that I want to use I created it on vmware
![image](https://user-images.githubusercontent.com/103559520/163116726-4e658744-3029-4c3f-94be-f33ddb196f8c.png)

**Comment by alkisg** [comment]
The easiest way is to recreate your VM image using MBR and a single ext4 partition.
Now it's trying to find your operating system in the first partition which is fat (EFI partition), and it fails.
It's possible to pass parameters so that LTSP knows that it should use the second partition instead, but it's easier to just use MBR/single ext4.

**Reply by Squialz** [reply]
it worked thank you but when i created the vm with a single partition in ext4 on virtualbox it didn't work but on vmware it was good

**Comment by alkisg** [comment]
VirtualBox also works fine, but you need to select the "vmdk, static" format.

**Reply by Squialz** [reply]
I did that and it still didn't work

**Comment by alkisg** [comment]
What was the error message?

**Reply by Squialz** [reply]
it's good the ltsp server works but for virtualbox there was no error message just a black screen no message and I waited 15 min and still nothing but too bad I did it via vmware and it works the server is ready so it's cool

**Comment by alkisg** [comment]
OK if you try it again, please mention what command you tried and saw black screen.
You ran `ltsp image ubuntu-flat.img` and the screen went blank after that command?!

**Reply by Squialz** [reply]
yes i ran this command
